src/wp-includes/class-wp-theme-json.php

```diff
@@ -2068,6 +2068,7 @@ protected static function flatten_tree( $tree, $prefix = '', $token = '--' ) {
 	 * @since 6.1.0 Added `$theme_json`, `$selector`, and `$use_root_padding` parameters.
 	 * @since 6.5.0 Output a `min-height: unset` rule when `aspect-ratio` is set.
 	 * @since 6.6.0 Passing current theme JSON settings to wp_get_typography_font_size_value().
+	 *              Process background properties using the Style Engine.
```

Comment:
I _think_ line breaks won't be picked up by the PHPDoc parser. I'd merge this with the previous line e.g.

```
@since 6.6.0 Pass current theme JSON settings to wp_get_typography_font_size_value() and process background properties.
```

Reply:
**TIL** Will update